lms/djangoapps/experiments/utils.py

```diff
@@ -37,6 +39,24 @@
     flag_name=u'add_programs',
     flag_undefined_default=False
 )
+
+# .. feature_toggle_name: experiments.add_program_price
```

Comment:
out of curiosity is this a new convention or something that we're following?

Reply:
It was a hackathon project that didn't get merged (as far as I know). I like the idea of documenting the intent of the flags.

I'm happy to change the format/content or remove it entirely if people want.